### PR TITLE
snap/quota,wrappers: support inheritting journal quota for service sub-groups

### PR DESCRIPTION
**Pull request by Meulengracht**
This support was missing for the per-service quotas. Since we no longer allow specifying journal quotas for service sub-groups, they should instead inherit their parent journal quota (if set). This adds this missing code.

### CODE REVIEW

File: wrappers/services_gen_test.go [hunk 1]
```diff
@@ -1019,3 +1021,158 @@ OOMScoreAdjust=-899
 WantedBy=multi-user.target
 `, mountUnitPrefix, mountUnitPrefix))
 }
+
+func (s *servicesWrapperGenSuite) TestQuotaGroupSlice(c *C) {
```
Comment (pedronis) [inline wrappers/services_gen_test.go:1025]: so there was no test for this before?

**Review by pedronis — APPROVED** [review]
lgtm

File: wrappers/services_gen_test.go (hunk 1 above)
Comment (Meulengracht) [reply wrappers/services_gen_test.go:1025]: Nope, it seems this was an oversight, so added one while I was down there, and thank you for taking time to get this looked at here on the last day before break!

File: wrappers/services_gen_test.go [hunk 2]
```diff
@@ -1019,3 +1021,158 @@ OOMScoreAdjust=-899
...
+		Snap: &snap.Info{
+			SuggestedName: "snap",
+			Version:       "0.3.4",
+			SideInfo:      snap.SideInfo{Revision: snap.R(44)},
+		},
+		Name:         "app",
+		Command:      "bin/foo start",
+		Daemon:       "simple",
+		DaemonScope:  snap.SystemDaemon,
+		RestartDelay: timeout.Timeout(20 * time.Second),
```
Comment (mvo5) [inline wrappers/services_gen_test.go:1036]: (nitpick) I guess this is also used in the other tests but it seems this setting is irrelevant for what we test so we could also just not have it here.

**Review by mvo5 — APPROVED** [review]
Thank you

File: wrappers/services_gen_test.go (hunk 2 above)
Comment (Meulengracht) [reply wrappers/services_gen_test.go:1036]: Nice catch, thank you! fixed

**Pull request merged**